torch_xla_py/xla_dist.py

```diff
@@ -473,26 +470,26 @@ def _add_docker_vars(self, cmd):
       self.container_name = 'pytorchtpudistrunner'
       container_name_cmd = '--name={}'.format(self.container_name)
 
-    docker_export_cmd = ' -e '.join([''] + DistributedExecutor.DIST_ENV_VARS)
-    idx = cmd.index(DistributedExecutor.DOCKER_RUN) \
-        + len(DistributedExecutor.DOCKER_RUN)
-    return '{} {} {} {}'.format(cmd[:idx], container_name_cmd,
-                                docker_export_cmd, cmd[idx:])
+    docker_export_cmd = ' -e '.join([''] + self.DIST_ENV_VARS)
+    idx = cmd.index(self.DOCKER_RUN) \
+        + len(self.DOCKER_RUN)
+    return ' '.join([cmd[:idx], container_name_cmd,
```

Comment:
by suggesting `.split(`, here I meant
```
cmd_head, cmd_tail = cmd.split(self.DOCKER_RUN)
return ' '.join(cmd_head, ..., cmd_tail)
```

but I guess this is fine too. My version assumes there's one `self.DOCKER_RUN` in the `cmd`.